### PR TITLE
develop

### PR DESCRIPTION
**Pull request by mrsvolodya**
https://mrsvolodya.github.io/js_sort_table_DOM/

### CODE REVIEW

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Well done, now pls fix [failing tests](https://github.com/mate-academy/js_sort_table_DOM/pull/898/checks#step:5:35)

**Comment by mrsvolodya** [comment]
Hi, I tried fix it but it don't fixed, my ESLint Failed in any way with this task,  screens below....
image
image

**Comment by mrsvolodya** [comment]
Can you advice how solve this misunderstanding behavior...

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
-// write code here
+const tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+
+const sortedList = (i) => {
+  const rows = Array.from(tbody.querySelectorAll('tr'));
+
+  if (i < 2) {
+    rows.sort((a, b) =>
+      a.children[i].textContent.localeCompare(b.children[i].textContent),
```
Comment (volodymyr-soltys97) [inline src/scripts/main.js:11]:
```suggestion
      a.children[i].textContent.localeCompare(b.children[i].textContent)
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, the linter now shows that the error is on line 12, another line is selected on your screen

**Pull request closed without merging**